src/controllers/usersController.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const { StatusCodes, ReasonPhrases } = require('http-status-codes');
+const usersService = require('../services/usersService');
+
+function get(_, res) {
+  res.status(StatusCodes.OK).send(usersService.get());
```

Comment:
The `get` function directly sends the result of `usersService.get()`. Consider adding error handling to manage cases where the service might return an error or an empty result.